scalafix/output/src/main/scala/fix/Collectionstrawman_v0.scala

```diff
@@ -1,6 +1,6 @@
 package fix
 
-import scala.Predef.{ augmentString => _, genericArrayOps => _, genericWrapArray => _, intArrayOps => _, wrapIntArray => _, wrapString => _, _ }
+import scala.Predef.{ augmentString => _, genericArrayOps => _, intArrayOps => _, wrapString => _, _ }
 import strawman.collection.{ arrayToArrayOps, stringToStringOps }
```

Comment:
I'm not pretty sure whether `genericWrapArray` and `wrapIntArray` should also be added. It was enough to include only `genericArrayOps` to make `output/../Collectionstrawman_v0_List` compile. Please let me know if it makes more sense to change the scalafix rule to unimport those 2 as well.

Reply:
In some tests I did, these conversions could also be picked. But if it does compile without unimporting them in that test, that’s fine. We might refine that behaviour later…